server/sqlflowserver.go

```diff
@@ -1,4 +1,8 @@
-//go:generate protoc -I proto proto/sqlflow.proto --go_out=plugins=grpc:proto
```

Comment:
Why should we remove this line? Should we remove [parser.go](https://github.com/sql-machine-learning/sqlflow/tree/develop/sql/parser.go) instead?